models/patch.models.js

```diff
@@ -0,0 +1,27 @@
+const db = require('../db/connection')
+const { validateArticleId } = require('../validation-functions/validateArticle-Id')
+
+async function updateVoteByArticleId (article_id, inc_votes) {
+   try {
+    
+   await validateArticleId(article_id)
```

Comment:
You shouldn't need a check here on article_id, in the query below if the article id is an invalid data type SQL will throw an error. If the query returns an empty array in the .then - you can infer that the article has not been found (article/99999)